pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -132,6 +132,12 @@ func (plugin *kubenetNetworkPlugin) Init(host network.Host, hairpinMode componen
 		return fmt.Errorf("Failed to generate loopback config: %v", err)
 	}
 
+	// Need to SNAT outbound traffic from cluster
+	// TODO: move this into cni bridge plugin
+	if _, err := plugin.iptables.EnsureRule(utiliptables.Append, utiliptables.TableNAT, utiliptables.ChainPostrouting, "-m", "comment", "--comment",
+		"kubenet: SNAT for outbound traffic from cluster", "-s", nonMasqueradeCIDR, "!", "-d", nonMasqueradeCIDR, "-j", "MASQUERADE"); err != nil {
```

Comment:
We used to periodically confirm this, now it's only on init, feels more brittle but I don't see an alternative. If you want to mimize margin for error, suggest invoking container_bridge:ensureIPTablesMasqRule.